shared/container.go

```diff
@@ -167,6 +167,10 @@ var KnownContainerConfigKeys = map[string]func(value string) error{
 
 	"linux.kernel_modules": IsAny,
 
+	"migration.pre_copy.enabled":            IsBool,
+	"migration.pre_copy.max":                IsUint32,
+	"migration.pre_copy.pre_migrated_pages": IsUint32,
```

Comment:
The keys should also be listed in the bash completion script.

Reply:
added